src/valid/analyzer.rs

```diff
@@ -181,6 +199,10 @@ pub struct FunctionInfo {
     ///
     /// Each item corresponds to an expression in the function.
     expressions: Box<[ExpressionInfo]>,
+    /// Vector with information of wether or not a functin argument is used for sampling
```

Comment:
"Vector with information of whether or not a function argument is used for sampling"